lib/common/actions/index.js

```diff
@@ -1,37 +1,44 @@
 import fetch from 'isomorphic-fetch'
+import {GTFS_GRAPHQL_PREFIX} from '../../common/constants'
 import { setErrorMessage } from '../../manager/actions/status'
 
-export function secureFetch (url, method = 'get', payload, raw) {
+export function secureFetch (url, method = 'get', payload, raw = false, isJSON = true) {
```

Comment:
I'm seeing a lot of `res.json().then(` in other areas.  Perhaps we can add something that assumes a response is json or assume that if `raw` is false, then json should be returned.